app/views/shared/processed_and_deleted/_index.html.slim

```diff
@@ -8,7 +8,7 @@
       table *{class: "#{state}-applications"}
         thead
           tr
-            th =t('processed_applications.table_header.applicant')
+            = table_header
```

Comment:
About the table column heading & rows, good catch. I was working on a outdated copy of the app, updating it now!